src/public/services/FieldVerificationService.ts

```diff
@@ -89,18 +89,21 @@ export const verifyOtp = async ({
 /**
  * Reset the field in the transaction, removing the previously saved signature.
  *
+ * @param formId The id of the form to reset the transaction for
  * @param transactionId The generated transaction id for the form
  * @param fieldId The id of the verification field to reset
  * @returns 200 OK status if successfully reset
```

Comment:
For clarity, should this be returning 200 or void

Reply:
the http status code is 200 but the request itself carries no information beyond that (and hence, `promise` returns void)!

Reply:
haven't reviewed yet besides this thread, but if 200 and nothing is returned, i think a better HTTP code is `201 Created`

Reply:
Oh but nothing is created, nvm

Reply:
the response code should be `204 ( No Content )` then!

Reply:
ah cool, didn't realise this! pasting the mdn links [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#successful_responses) for future readers!